app/index.html

```diff
@@ -23,9 +23,8 @@
       <script src="bower_components/angular-ui-router/release/angular-ui-router.js"></script>
       <script src="bower_components/angular-resource/angular-resource.js"></script>
       <script src="app.js"></script>
-      <script type="text/javascript" src="index.config.js"></script>
-      <script type="text/javascript" src="index.constant.js"></script>
-      <script type="text/javascript" src="common/interceptor.js"></script>
+      <script type="text/javascript" src="components/outlets/outletController.js"></script>
+      <script type="text/javascript" src="directives/tableDirective.js"></script>
       <link rel="stylesheet" type="text/css" href="common/styles.css">
```

Comment:
sir that time i had make a directive to render the table heading on the page now i have used styles too.